.github/workflows/site.yaml

```
@@ -57,6 +57,8 @@ jobs:
           ./.github/bin/build_docs.sh 'tags/v0.2.1-incubating.tar.gz' '0.2.1-incubating'
       - run: |
           ./.github/bin/build_docs.sh 'tags/v0.3.2-incubating.tar.gz' '0.3.2-incubating'
+      - run: |
+          ./.github/bin/build_docs.sh 'tags/v0.4.0-incubating.tar.gz' '0.4.0-incubating'
```

inline it and above steps
```suggestion
      - run: ./.github/bin/build_docs.sh 'tags/v0.4.0-incubating.tar.gz' '0.4.0-incubating'
```